packages/clay-color-picker/src/Custom.tsx

```diff
@@ -42,6 +42,26 @@ const RGBInput: React.FunctionComponent<IRGBInputProps> = ({
 	const inputRef = React.useRef(null);
 	const [inputValue, setInputValue] = React.useState(value);
 
+	const handleOnChange = (event: any) => {
+		const value = event.target.value;
+
+		if (value === '') {
+			return;
+		}
+
+		let newVal = Number(value);
+
```

Comment:
This looks OK to me, but I was wondering why we weren't using `parseInt` 

Why not use?

```js
let newVal = parseInt(value, 10);`
```

Is there anything better about using the `Number` constructor (without `new`) to do the cast?
(Non blocking comment)

Reply:
I think it's the same, I kept previous behavior, don't know if there's any advantage on any of them.

Reply:
Got it.

I know that there is a difference,  for example

```js
parseInt("");  
```
Will return NaN

And 

```js
Number(""); 
```
Will return 0

Reply:
Then we should keep `Number`, 'cause if `""` is received as value (that happens when you enter a non-numeric character in an number input) we `return` to avoid any update.

Reply:
Well, you could still do the same with `parseInt`, since you're returning before casting the string to a number, right?

Reply:
Anyway, I don't think it makes much of a difference, I just personally don't like using constructors without `new` but the result is going to be the same.

Thanks for the interesting conversation Carlos :smile: 

I'll let the clay experts do the review :+1:

Reply:
> I'll let the clay experts do the review 👍

![image](https://user-images.githubusercontent.com/6843530/109202056-66790780-7757-11eb-82de-1980d6f17b0b.png)